python/pyspark/sql/dataframe.py

```diff
@@ -667,6 +667,51 @@ def repartition(self, numPartitions, *cols):
         else:
             raise TypeError("numPartitions should be an int or Column")
 
+    @since("2.3.0")
```

Comment:
Hm, I prefer 2.4.0 but am okay with 2.3.0.

Reply:
Sorry, shall we just target 2.4.0?

Reply:
ok, rename to 2.4.0, done